features/support/world.ts

```diff
@@ -38,10 +38,9 @@ export class World extends CucumberWorld {
 
   public print (message: string): void {
     if (this.scenarioRunning) {
-      this.log(message) as any
-      // console.log(message)
+      void this.attach(message)
     } else {
-      // console.log(message)
+      // this.attach(message)
```

Comment:
`this.log` just prints `Attachment (text/x.cucumber.log+plain)` as the cucumber printer for some reason doesn't know about the cucumber log mime type. `this.attach` actually prints the string in the log instead.